src/fuel-vm/index.md

```diff
@@ -25,6 +25,9 @@ This document provides the specification for the Fuel Virtual Machine (FuelVM).
 | `MEM_MAX_ACCESS_SIZE`   | `uint64` |         | Maximum memory access size, in bytes.   |
 | `VM_MAX_RAM`            | `uint64` | `2**26` | 64 MiB.                                 |
 | `MESSAGE_MAX_DATA_SIZE` | `uint64` |         | Maximum size of message data, in bytes. |
+| `MAX_INPUTS`            | `uint16` | `256`   | Maximum number of inputs.               |
```

Comment:
Hmm, we are using consensus parameters inside of the VM. And we have these fields there https://github.com/FuelLabs/fuel-specs/blob/master/src/tx-format/consensus_parameters.md

Maybe we need to use them here instead

Reply:
True. Removed these in 1227f77